scss/_variables.scss

```diff
@@ -779,6 +779,7 @@ $modal-footer-border-color:   $modal-header-border-color !default;
 $modal-header-border-width:   $modal-content-border-width !default;
 $modal-footer-border-width:   $modal-header-border-width !default;
 $modal-header-padding:        1rem !default;
+$modal-border-radius:         $border-radius-lg !default;
```

Comment:
Hi @akolinski thanks a lot for this PR. I've tested it and it works.

Since this variable is affecting the `.modal-content` I'd move the variable up to line 776 and name it `$modal-content-border-radius`.